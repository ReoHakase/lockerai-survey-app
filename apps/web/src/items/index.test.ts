import { describe, it, expect } from 'vitest';
import { getImageIds } from '.';

describe('getImageIds', () => {
  it('JSONファイルからラベルがついている画像のIDを全て取得できる', () => {
    expect(getImageIds()).matchSnapshot();
  });
});
