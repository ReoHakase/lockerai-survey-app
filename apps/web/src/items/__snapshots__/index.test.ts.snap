// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`getImageIds > JSONファイルからラベルがついている画像のIDを全て取得できる 1`] = `
[
  "00299661bf57253d",
  "0ea2faa8f80d09d2",
  "10442d366d5061ee",
  "1a4df54f9ecec3c6",
  "1a6a82e6106c75a6",
]
`;
